src/ensembl/src/shared/components/visibility-icon/VisibilityIcon.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { VisibilityIcon } from './VisibilityIcon';
+import { Status } from 'src/shared/types/status';
+
+describe('renders', () => {
+  it('partially selected icon ', () => {
```

Comment:
1) usually, a good practice is to write the description for the test so that it reads as a sentence when read alongside the function name.
`describe('rendering')`
`it('renders partially selected icon')`
`test('button visibility')` (does the same as `it`)

2) Test description says something about the icon, but the actual code of the test checks the class. I am not sure what the test was intended to test, but either the description, or the implementation are lying.

Reply:
Thanks Andrey I have changed the description to match the test. Test is to check it it applies proper class when button status is PARTIALLYSELECTED.